src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -492,6 +493,168 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         self.name = self.m_def.name if self.name is None else self.name
 
 
+class ParticleCell(Cell):
+    """
+    A base section used to specify the atomic cell information of a system.
+    """
+
+    particles_state = SubSection(sub_section=ParticlesState.m_def, repeats=True)
+
+    n_particles = Quantity(
+        type=np.int32,
+        description="""
+        Number of atoms in the atomic cell.
+        """,
+    )
+
+    equivalent_particles = Quantity(
+        type=np.int32,
+        shape=['n_atoms'],
+        description="""
+        List of equivalent atoms as defined in `atoms`. If no equivalent atoms are found,
+        then the list is simply the index of each element, e.g.:
+            - [0, 1, 2, 3] all four atoms are non-equivalent.
+            - [0, 0, 0, 3] three equivalent atoms and one non-equivalent.
+        """,
+    )
+
+    def __init__(self, m_def: 'Section' = None, m_context: 'Context' = None, **kwargs):
+        super().__init__(m_def, m_context, **kwargs)
+        # Set the name of the section
+        self.name = self.m_def.name  #! self.name here is 'ParticleCell'
+
+    def is_equal_cell(self, other) -> bool:
```

Comment:
What exactly is this used for in atomic_cell? Is it for defining different reference frames?